src/jadhav_lab_to_nwb/olson_2024/olson_2024_sorting_interface.py

```diff
@@ -0,0 +1,62 @@
+"""Primary class for converting experiment-specific spike sorting."""
+from pynwb.file import NWBFile
+from pydantic import DirectoryPath
+from pathlib import Path
+import pandas as pd
+from natsort import natsorted
+from neuroconv.basedatainterface import BaseDataInterface
+
+
+class Olson2024SortingInterface(BaseDataInterface):
+    """Sorting interface for olson_2024 conversion"""
+
+    keywords = ("extracellular electrophysiology", "spike sorting")
+
+    def __init__(self, spike_times_folder_path: DirectoryPath, unit_stats_folder_path: DirectoryPath):
+        super().__init__(spike_times_folder_path=spike_times_folder_path, unit_stats_folder_path=unit_stats_folder_path)
+
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
+        spike_times_folder_path = Path(self.source_data["spike_times_folder_path"])
+        unit_stats_folder_path = Path(self.source_data["unit_stats_folder_path"])
+        nwbfile.add_unit_column(name="nTrode", description="The tetrode number for this unit")
+        nwbfile.add_unit_column(name="unitInd", description="The integer id each unit within each nTrode")
+        nwbfile.add_unit_column(
+            name="global_id", description="The global id for each unit: nTrode{nTrode}_unit{unitInd}"
+        )
+        nwbfile.add_unit_column(name="nWaveforms", description="Number of waveforms (spikes) for each unit.")
+        nwbfile.add_unit_column(
+            name="waveformFWHM", description="Full width at half maximum of the template waveform in ms."
+        )
+        nwbfile.add_unit_column(
+            name="waveformPeakMinusTrough", description="Peak minus trough of the template waveform in uV."
+        )
+
```

Comment:
Minor comment: I noticed column names are written using camel case, but `global_id` in snake case. Is that intentional?